uiowa/uiowa
# Feature: final pdf updates

**Remaining tasks:**
- [x] Compare D7 pdf fields with D8

# How to test

- `blt ds --site=admissions.uiowa.edu`
- `yarn workspace admissions_core build`
- `drush @admissions.local cr`
- Go to http://admissions.local.drupal.uiowa.edu/areas-of-study/print-links
- Verify that you can wrap a section in the text area with `<div class="print__sidebar--right"></div>` and the content will render in the pdf as a column floated to the right with a gray background.  

<img width="994" alt="Screen Shot 2021-02-26 at 3 57 03 PM" src="https://user-images.githubusercontent.com/1036433/109359555-4cb8ec80-784b-11eb-8df3-1ee427b70d90.png">

- Verify that you can wrap multiple `<ul>` in `<div class="print__list--column"></div>` and they will stack in a column like:

<img width="954" alt="Screen Shot 2021-02-26 at 3 56 40 PM" src="https://user-images.githubusercontent.com/1036433/109359527-3d39a380-784b-11eb-802d-1a5b924f0cd8.png">

- Verify that the `Roboto Bold` font is working as it appears to the example below
- Verify that the Certificates appear without a checkbox as it appears to the example below

<img width="1016" alt="Screen Shot 2021-02-26 at 3 45 37 PM" src="https://user-images.githubusercontent.com/1036433/109359098-7a516600-784a-11eb-81d8-49e937654b57.png">

<details>
Possible help text
For the print version, the following adjustments can be made:
<ul>
<li>Pull a block of text out and display it on the right side of the page in a gray box by wrapping the content in `<div class="print__sidebar--right"></div>` </li>
<li>Force a set of lists to display side-by-side by wrapping each one in `<div class="print__list--column"></div>` </li>
</ul>
</details>
